scripts/stubtest/allowlist.txt

```diff
@@ -524,3 +524,8 @@ django.db.models.fields.generated.GeneratedField.__init__
 # https://github.com/python/cpython/blob/5abff6960b4aecb0d5c81c7482cf3faa74e1983d/Lib/copyreg.py#L112-L161
 # Some Django classes get copied at import time, which leads stubtest to detect it.
 .*\.__slotnames__
+
+# Depends on availability of geoip2.database
+django.contrib.gis.geoip2.__all__
```

Comment:
```suggestion
```

Reply:
It is always available in our CI.

Reply:
I already described the problem things in the PR message. Are you suggesting to set `__all__` as in CI?

PS: judging by `__all__` that I have, then `geoip2.database` is not available in our CI

Reply:
> vars `django.contrib.gis.geoip2.__all__` and `django.contrib.gis.utils.__all__` are dynamic so i put them to allowlist

Reply:
I mean that CI is failing right now: see https://github.com/typeddjango/django-stubs/actions/runs/14650357922/job/41119387542?pr=2624#step:6:30

We need to remove the unused allowlist entry.

Reply:
got you. should i also set `django.contrib.gis.utils.__all__` like in our CI and remove it from allowlist?

Reply:
yes, please :)

Reply:
the CI is the source of truth for us